e2e/tests/nightly/ras-login.spec.ts

```diff
@@ -94,6 +97,19 @@ describe('RAS Test', () => {
     const submitButton = await newPage.waitForXPath('//input[@value="Submit" and @type="submit"]', { visible: true });
     await submitButton.click();
 
+    // Could show up
+    try {
+      const maybeAgreeAndContinueButton = await newPage.waitForXPath(
+        '//input[@value="Agree and continue" and @type="submit"]',
+        {
+          visible: true
+        }
+      );
+      await maybeAgreeAndContinueButton.click();
+    } catch (err) {
+      // Blank
```

Comment:
log on error?

Reply:
Nah. I'm assuming button not exists. so not logging err here.